### ISSUE TITLE
Missing assets

### ISSUE THREAD

**Issue by marvoh** [takielias/tablar#63]
Hello,

Thank you again for your wonderful efforts.

The login page doesn't seem to be loading assets. Do you have an idea why?

Please see errors below:

![image](https://github.com/takielias/tablar/assets/5451142/c2c21422-1426-43bd-bff7-264319e0c8da)

Best,
N

**Comment by takielias** [comment]
@marvoh Did you follow the steps? 

![image](https://github.com/takielias/tablar/assets/38932580/1ccf51f5-3b3e-4d8c-ba4d-ed1fe35e76e7)

**Comment by marvoh** [comment]
Yes, I followed all those steps including stopping and starting the npm command like you did on your YouTube video but without success

**Comment by takielias** [comment]
@marvoh Could you please share your dev environment? 

PHP version:
Nodejs version:
NPM version:
Laravel Version:

**Comment by marvoh** [comment]
The problem is a dependency issue. 

When I run ```npm install``` the following is the result:

![image](https://github.com/takielias/tablar/assets/5451142/93e1ef4d-b2ae-4ece-99cc-3632b8036949)

A quick workaround is to run ```yarn install``` instead which is much better at resolving dependencies.

Laravel version: v11.5.0
PHP version: 8.3.6
Nodejs version: 20.12.2
NPM version: 10.6.0

Thank you for your quick response and desire to fix issues that we encounter from time to time

**Comment by takielias** [comment]
@marvoh Could you try on Nodejs 16/18?

**Comment by marvoh** [comment]
I am sorry I haven't managed to try this but I think it could be the problem. 

I'm wondering also whether it could be related to a new issue I am facing.

I have included the data table as follows in my blade template ```<x-tabulator :table="$table"></x-tabulator>``` and it renders almost okay when there is no data.

When I add data to the database, I realise that the assets (css / js) are not loaded automatically. Is this the expected behaviour therefore I add them manually?

Thank you again for the good work.

**Comment by marvoh** [comment]
I'm sorry I have not managed to test it. I will let you know how it goes but it's currently working with yarn

**Comment by takielias** [comment]
@marvoh According to your scenario, it seems ok from the package end. Maybe it's a nodejs issue.
